coremltools/converters/nnssa/coreml/ssa_converter.py

```diff
@@ -36,6 +36,7 @@ def ssa_convert(ssa,
                 inputs=None,
                 outputs=None,
                 image_input_names=None,
+                image_format=None,
```

Comment:
@aseemw Do you think we need to provide default values here?

Reply:
What's the behavior if provided image_format and inferred image format mismatch?

Reply:
As I see it the default `None` is being used to determine automatically from the graph the format. otherwise the user can override.

Reply:
that's correct. I can add some documentation to make that more clear.